mesonbuild/modules/features/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023, NumPy Developers.
+
```

Comment:
I'd like to see `future.__annotations__` used as well, so we don't have to quote all of the argument types.